clients/fides-js/src/lib/tcf/fidesString.ts

```diff
@@ -0,0 +1,39 @@
+import { FIDES_SEPARATOR } from "./constants";
+import { VendorSources } from "./vendors";
+
+export const decodeFidesString = (fidesString: string) => {
+  const split = fidesString.split(FIDES_SEPARATOR);
+  if (split.length === 1) {
+    return { tc: split[0], ac: "" };
+  }
+  if (split.length >= 2) {
+    const [tc, ac] = split;
+    if (tc === "") {
+      return { tc: "", ac: "" };
+    }
+    return { tc, ac };
+  }
+  return { tc: "", ac: "" };
+};
+
+/**
+ * Given an AC string, return a list of its ids, encoded
+ *
+ * @example 1~1.2.3 --> [gacp.1, gacp.2, gacp.3]
```

Comment:
Nice! What does the `@example` annotation do exactly?

Reply:
oh heh I think I had just seen it around before, but only just looked up the documentation: https://jsdoc.app/tags-example.html

so it actually looks like it should be in the format of calling the function to show it being used as an example... I'll update the docstring, but basically it formats the following line to look like code

<img width="490" alt="image" src="https://github.com/ethyca/fides/assets/24641006/c05722e8-563f-417d-abdf-c39ec9322957">